spanner/src/session.rs

```diff
@@ -233,12 +233,23 @@ impl SessionPool {
     ) -> Result<VecDeque<SessionHandle>, Status> {
         let channel_num = conn_pool.num();
         let creation_count_per_channel = min_opened / channel_num;
+        let remainder = min_opened % channel_num;
 
         let mut sessions = Vec::<SessionHandle>::new();
+        let mut tasks = JoinSet::new();
         for _ in 0..channel_num {
+            // Ensure that we create the exact number of requested sessions by adding the remainder to the first channel.
```

Comment:
Also took the opportunity to implement the same exact session creation as the golang client: https://github.com/googleapis/google-cloud-go/blob/7cbffad749a0f8917658406dae73c7fb7151b55d/spanner/sessionclient.go#L221

Ran into this during testing when creating a client where `channel_num > min_opened`.